pkgs/tools/networking/bacnet-stack/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "bacnet-stack";
+  version = "2019-04-14";
+
+  src = fetchgit {
+    url = "https://github.com/stargieg/bacnet-stack.git";
```

Comment:
Why from stargieg/bacnet-stack?

Seems to me the newest version is https://github.com/bacnet-stack/bacnet-stack

Reply:
Good catch! I'm not sure how I googled my way to that other repo.